src/vc_util/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "vc_util"
+description = "Utils for verifiable credentials on the IC"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+# ic dependencies
+candid = "0.9"
+ic-certified-map = "0.4"
+ic-crypto-standalone-sig-verifier = { git = "https://github.com/dfinity/ic", rev = "bf9bc00b032a18d2dbcfcfcfb7a76a562f350c9e" }
+ic-types = { git = "https://github.com/dfinity/ic", rev = "bf9bc00b032a18d2dbcfcfcfb7a76a562f350c9e" }
+canister_sig_util = { path = "../canister_sig_util" }
+
+# vc dependencies
+identity_core = { git = "https://github.com/frederikrothenberger/identity.rs.git", branch = "frederik/wasm-test", default-features = false }
+identity_credential = { git = "https://github.com/frederikrothenberger/identity.rs.git", branch = "frederik/wasm-test", default-features = false , features = ["validator"] }
+identity_jose = { git = "https://github.com/frederikrothenberger/identity.rs.git", branch = "frederik/wasm-test", default-features = false, features = ["iccs"]}
```

Comment:
Are there official repos we can link to? also I think that particular dep creates a circular dependency to an older commit of II

Reply:
No, we don't have an official repo for that yet.
There is no circular dependency, as I've removed recently the dependency of `identity.rs`-crates on II-repo.